src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1160,23 +1161,64 @@ private void NextScheduledActivityCallback(string databaseName, IdleDatabaseActi
                             break;
 
                         case IdleDatabaseActivityType.WakeUpDatabase:
-                            _ = TryGetOrCreateResourceStore(databaseName, nextIdleDatabaseActivity.DateTime).ContinueWith(t =>
+                            if (_serverStore.ConcurrentBackupsCounter.CanRunBackup == false)
                             {
-                                var ex = t.Exception.ExtractSingleInnerException();
-                                if (ex is DatabaseConcurrentLoadTimeoutException e)
+                                // reached max concurrent backups, retry after 1 min
+
+                                if (_logger.IsInfoEnabled)
+                                    _logger.Info($"Delaying the start of the database '{databaseName}' for running a backup because we reached max concurrent backups, will retry the wakeup in {_dueTimeOnRetry:#,#;;0}ms");
+
+                                RescheduleDatabaseWakeup();
+                                break;
+                            }
+
+                            if (CanServerRunBackup() == false)
+                            {
+                                // the server cannot run the backup anyway (low memory, low cpu credits or high dirty memory state)
+
+                                if (_logger.IsInfoEnabled)
+                                    _logger.Info($"Delaying the start of the database '{databaseName}' for running a backup because we are in a low memory state, will retry the wakeup in {_dueTimeOnRetry:#,#;;0}ms");
+
+                                RescheduleDatabaseWakeup();
+                                break;
+                            }
+
+                            var startDatabaseForBackup = _serverStore.ConcurrentBackupsCounter.TryStartDatabaseForBackup();
+                            if (startDatabaseForBackup == null)
+                            {
+                                // reached max concurrent loading of databases for backup, retry after 1 min
+
+                                if (_logger.IsInfoEnabled)
+                                    _logger.Info($"Delaying the start of the database '{databaseName}' for running a backup because we reached max concurrent loading of databases for backup, will retry the wakeup in {_dueTimeOnRetry:#,#;;0}ms");
```

Comment:
also write the value here, how many databases can be loaded for backups concurrently

Reply:
done